docs/core/extensions/options.md

```diff
@@ -27,8 +27,8 @@ Create the following `TransientFaultHandlingOptions` class:
 
 An options class:
 
-- It allows for either parameterless constructors, or for constructors where all parameters have default values
-- Properties can be read-write, or read-init only
+- Must be a non-abstract class with either a public parameterless constructor, or a public constructor where all parameters have default values.
```

Comment:
You added "class" here. Is this correct? Or can structs also be used for options?

Reply:
Also, records.  Positional records won't work because of the constructor requirement.

Reply:
>  Positional records won't work because of the constructor requirement.

Are you sure about this? Aren't positional records something like this (which is the example given in the docs):

```c#
public record TransientFaultHandlingOptions(bool Enabled = false, TimeSpan AutoRetryDelay = default);
```

Reply:
Positional records do actually work, I have tested that locally. What I have not done was exhaustively tested all variants of access modifier and type, i.e.; `struct`, `readonly struct`, `tuple`, `interface`, etc. I'm assuming that there is some generic type parameter constraint, but I haven't dug into that yet. What we have thus far is technically still accurate, just might be missing something.